src/main/java/Babe.java

```diff
@@ -0,0 +1,448 @@
+                    chatBot.sayBye();
+                    
+                } else if (instruction.equalsIgnoreCase("list") && inputLength == 1) {
+                    chatBot.printList();
+                    
+                } else if (instruction.equalsIgnoreCase("mark")) {
+                    chatBot.changeStatus(true);
+                    chatBot.save();
+                    
+                } else if (instruction.equalsIgnoreCase("unmark")) {
+                    chatBot.changeStatus(false);
+                    chatBot.save();
+                    
+                } else if (instruction.equalsIgnoreCase("todo")) {
+                    if (inputLength == 1) {
+                        throw new NoDescriptionException();
+                    }
+                    chatBot.addToDo(chatBot.rebuildUserInput(1, inputLength), true);
+                    chatBot.save();
+                    
+                } else if (instruction.equalsIgnoreCase("deadline")) {
+                    if (inputLength == 1) {
+                        throw new NoDescriptionException();
+                    }
+
+                    int deadlineIndex = chatBot.findArgument("/by");
+                    String deadline = "";
+
+                    try {
+                        deadline = chatBot.formatDate(chatBot.rebuildUserInput(deadlineIndex,
+                                inputLength));
+                        chatBot.addDeadline(chatBot.rebuildUserInput(1, deadlineIndex - 1),
+                                deadline,
+                                true);
+                        chatBot.save();
+                    } catch (Exception e) {
+                        Babe.drawLine();
+                        System.out.println("The date format should be yyyy-mm-dd, luv. Please try again.");
+                        Babe.drawLine();
+                    }
+
+
+                } else if (instruction.equalsIgnoreCase("event")) {
+                    if (inputLength == 1) {
+                        throw new NoDescriptionException();
+                    }
+
+                    int startDateIndex = chatBot.findArgument("/from");
+                    int endDateIndex = chatBot.findArgument("/to");
+                    String startDateAndTime = "";
+                    String endDateAndTime = "";
+
+                    try {
+                        startDateAndTime = chatBot.formatDate(chatBot.rebuildUserInput(startDateIndex,
+                                endDateIndex - 1));
+                        endDateAndTime = chatBot.formatDate(chatBot.rebuildUserInput(endDateIndex,
+                                inputLength));
+                        chatBot.addEvent(chatBot.rebuildUserInput(1, startDateIndex - 1),
+                                startDateAndTime,
+                                endDateAndTime,
+                                true);
+                        chatBot.save();
+
+                    } catch (Exception e) {
```

Comment:
May be a better practice to catch the specific types of Exceptions, instead of catching the general Exception